alpaca.sh

```diff
@@ -12,6 +12,7 @@ env:
 #!/bin/sh
 rsync -a $SOURCEDIR/* .
 make
+cp -r * $INSTALLROOT
```

Comment:
It's quite possible that there is a `make install` target, which would avoid copying unnecessary files. Please use that one if at all possible!

Reply:
Yes, you suggested me this before. However, it says there is no rule to make it;

```make: *** No rule to make target `install'.  Stop.```

Reply:
In that case, I would still suggest only copying headers and binary files (alongside possibly generated documentation and other files you might need for the generation).